packages/odyssey-react/src/components/Select/Select.module.scss

```diff
@@ -323,7 +323,8 @@
     font-size: $size-body-base;
     font-weight: 400;
     line-height: calc((#{$base-line-height} * 1em) - 2px);
-    // Adjust for pill border
+
+    /* Adjust for pill border */
```

Comment:
reworking this file to use standard CSS comments to avoid additional parser complexity for our tooling